woocommerce-services.php

```diff
@@ -500,6 +500,10 @@ public function init_core_wizard_payments_config() {
 				if ( is_wp_error( $response ) ) {
 					// TODO handle case of existing account
 					$this->logger->debug( $response, __CLASS__ );
+				} else {
+					$stripe_settings = get_option( 'woocommerce_stripe_settings', array() );
+					$stripe_settings['connect'] = 'yes';
+					update_option( 'woocommerce_stripe_settings', $stripe_settings );
```

Comment:
What do you think about doing this in [`WC_Connect_Stripe::save_stripe_keys()`](https://github.com/Automattic/woocommerce-services/blob/master/classes/class-wc-connect-stripe.php#L84) instead?